lib/compute-download-urls.js

```diff
@@ -231,6 +240,36 @@ function getWindowsFirefoxDriverArchitecture(version, wantedArchitecture) {
   return 'win.zip';
 }
 
+function getEdgeDriverDirectory(version) {
+  var release = parseInt(version)
+  switch (release) {
+    case 15063:
+      return '3/4/2/342316D7-EBE0-4F10-ABA2-AE8E0CDF36DD'
+    case 14393:
+      return '3/2/D/32D3E464-F2EF-490F-841B-05D53C848D15'
+    case 10586:
+      return 'C/0/7/C07EBF21-5305-4EC8-83B1-A6FCC8F93F45'
+    case 10240:
+      return '8/D/0/8D0D08CF-790D-4586-B726-C6469A9ED49C'
+    default:
+      throw new Error('Invalid Microsoft Edge Web Driver version see https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/#downloads')
+  }
+}
+
+function getEdgeDriverExtension(version) {
+  var release = parseInt(version)
+  switch (release) {
+    case 15063:
+    case 14393:
+      return 'exe'
+    case 10586:
+    case 10240:
+      return 'msi'
```

Comment:
I see that these two versions are released as an `.msi` package, but there isn't any code to run that installer and get the path to the `MicrosoftWebDriver.exe` file that we really need.

You can look at how this post-download is handled for Firefox which is shipped as a `.zip` or `.tar.gz`. https://github.com/vvo/selenium-standalone/blob/baf462261ae86d401f369d1c76542fcb3b7be3af/lib/install.js#L186